packages/edit-post/src/components/layout/style.scss

```diff
@@ -1,69 +1,107 @@
-$resize-handle-height: $grid-unit-30;
-
 .edit-post-meta-boxes-main {
 	filter: drop-shadow(0 -1px rgba($color: #000, $alpha: 0.133)); // 0.133 = $gray-200 but with alpha.
+	// Windows High Contrast mode will show this outline, but not the shadow.
+	outline: 1px solid transparent;
 	background-color: $white;
 	clear: both; // This is seemingly only needed in case the canvas is not iframe’d.
```

Comment:
Thanks for noticing this! This rule seems like it should be preserved but it was in the wrong place. 89fdd4edb12e8949086297582844deb7e6f2281b moves it so it will actually apply. I just realized, I could have specified it to not apply when the iframe is present, but the rule isn’t harmful and it costs more bytes write the longer selector.